### PR TITLE
docs: more 8.6 release note reqs + breaking changes

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

- [x]  Adds 8.6 release note syntax.
- [x]  Adds breaking changes information. Content sourced from Andrew's PR descriptions with minor changes and maybe an assumption or two, so please give them a read 😄.

### CODE REVIEW

File: docs/apm-breaking.asciidoc [hunk 1]
```diff
@@ -5,14 +5,70 @@
...
+For more details, see https://github.com/elastic/apm-server/pull/9565[PR #9565].
+
+[float]
+==== `apm-server apikey` commands deprecated
+The `apm-server apikey` commands have been deprecated.
+API keys should be managed through either the Kibana or Elasticsearch REST APIs;
+see <<api-key>> to learn how.
+
+For more details, see https://github.com/elastic/apm-server/pull/9446[PR #9446].
+// end::86-bc[]
```
Comment (simitt) [inline docs/apm-breaking.asciidoc:70]: Are we usually listing `deprecations` as breaking changes? It's specifically not a breaking change but only deprecated.

File: docs/apm-breaking.asciidoc [hunk 2]
```diff
@@ -5,14 +5,70 @@
...
+[float]
+==== `ecs.version` is now a `constant_keyword` field
+Instead of adding `ecs.version` as a field in every document written by APM Server,
+it is now a `constant_keyword` field with its value set in data stream mappings.
+
+This change removes `ecs.version` from `_source`. It is now only available through
+the {ref}/script-fields-api.html[{es} `field` API].
+APM does not currently rely on the `ecs.version` field, so there should be no noticeable change.
+
+For more details, see https://github.com/elastic/apm-server/pull/9208[PR #9208].
```
Comment (simitt) [inline docs/apm-breaking.asciidoc:39]: IMO this can really be considered a technical detail, not sure how this would imact anyone.

**Review by simitt — COMMENTED** [review]
Generally looks good, left some small comments

File: docs/apm-breaking.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/apm-breaking.asciidoc:70]: Good point. After looking around, no we do not. I'll remove this.

File: docs/apm-breaking.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/apm-breaking.asciidoc:39]: Perfect. I'll remove it.

**Comment by bmorelli25** [comment]
@reakaleek I wonder if https://github.com/elastic/apm-server/pull/9812 is working as expected? This PR only touches `.asciidoc` files, but it looks like all of the ci checks are running.

**Comment by bmorelli25** [comment]
Okay, maybe I could use your help, @reakaleek. The `system-test` check seems to be stuck in the `Waiting for status to be reported` stage on all of my doc-only PRs.

**Pull request merged**